script/postrelease

```diff
@@ -1,42 +1,45 @@
-#!/usr/bin/env ruby
-require 'json'
-
-def verbose command
-  puts("--> #{command}") || system(command) || fail("Failed: #{command}")
-end
-
-Dir.chdir(File.expand_path(File.join(File.dirname(__FILE__), '..')))
-
-puts
-puts "Enter the version to deploy to prod"
-puts
-npm_version = STDIN.gets.strip
-puts "Are you sure to deploy '#{npm_version}' this to prod?"
-puts "Current staged and local changes will be lost!"
-puts
-puts "Type 'yes' to continue"
-x = STDIN.gets
-exit(1) unless x.strip == 'yes'
-puts
-
-verbose('git fetch origin')
-verbose('git clean -ffxd .')
-
-verbose("git checkout tags/v#{npm_version} -b deploy-to-prod")
-Dir.chdir('demos/browser/')
-verbose('npm uninstall amazon-chime-sdk-js')
-verbose("npm install amazon-chime-sdk-js@#{npm_version}")
-
-demo_package = JSON.parse(File.read('package.json'))
-
-if demo_package['dependencies']['amazon-chime-sdk-js'] != "^#{npm_version}"
-  puts 'npm version is not the same as the version specified'
-  exit(1)
-end
-
-verbose('npm run build')
-verbose('npm run build --app=meetingReadinessChecker')
-
-Dir.chdir('../serverless/')
-verbose('npm run deploy -- -b chime-sdk-demo-prod-canary -o chime-sdk-demo-prod-canary -s chime-sdk-demo-prod-canary -u false -t -l')
-verbose('npm run deploy -- -b chime-sdk-meeting-readiness-checker-prod-canary -s chime-sdk-meeting-readiness-checker-prod-canary -a meetingReadinessChecker -u false -t -l')
+#!/bin/bash
+set -euxo pipefail
```

Comment:
Well this is basically the same as the deploy-canary-demo script now. Can we just delete this and move what you added to that script?

Reply:
Yes, we can do that will remove postrelease file